checks/dangerous_workflow.go

```diff
@@ -167,9 +167,9 @@ func checkJobForUntrustedCodeCheckout(job *actionlint.Job, path string,
 			continue
 		}
 		if strings.Contains(ref.Value.Value, "github.event.pull_request") {
-			line := 1
+			line := checker.OffsetDefault
 			if step.Pos != nil {
-				line = step.Pos.Line
+				line = uint(step.Pos.Line)
 			}
```

Comment:
These 4 lines can be replaced with `fileparser.GetLineNumber(step.Pos)`. Similar on line 228 in this file.

Reply:
thanks, will do!